apps/admin/src/components/users/details/consumption/user-consumption-aggregation-option.tsx

```diff
@@ -1,9 +1,19 @@
-"use client";
-
+import React, { useMemo } from "react";
```

Comment:
Wieso wurde "use client" hier entfernt? Wir benutzen ja useMemo etc.

Klar wird es behoben wenn es in einem Client Componentn integriert wurde aber trotzdem?